react/src/components/ComputeSessionNodeItems/AppLauncherModal.tsx

```diff
@@ -0,0 +1,345 @@
+                              src={app?.src}
+                              alt={app?.name}
+                              preview={false}
+                              style={{ height: 36, width: 36 }}
+                            />
+                          }
+                          onClick={() => {
+                            handleAppLaunch(app);
+                          }}
+                          style={{ height: 72, width: 72 }}
+                        />
+                        <Typography.Text style={{ textAlign: 'center' }}>
+                          {app?.title}
+                        </Typography.Text>
+                      </Flex>
+                    </Col>
+                  );
+                })}
+              </Row>
+            </div>
+          );
+        })}
+        {preOpenAppTemplate.length > 0 ? (
+          <>
+            <Typography.Title level={5} style={{ marginTop: 0 }}>
+              {t('session.launcher.PreOpenPortTitle')}
+            </Typography.Title>
+            <Row gutter={[12, 12]}>
+              {_.map(preOpenAppTemplate, (app) => {
+                return (
+                  <Col
+                    key={app?.name}
+                    span={6}
+                    style={{ alignContent: 'center' }}
+                  >
+                    <Flex
+                      direction="column"
+                      gap={'xs'}
+                      style={{ height: '100%' }}
+                    >
+                      <Button
+                        icon={
+                          <Image
+                            src={app?.src}
+                            alt={app?.name}
+                            preview={false}
+                            style={{ height: 36, width: 36 }}
+                          />
+                        }
+                        onClick={() => {
+                          handleAppLaunch(app);
+                        }}
+                        style={{ height: 72, width: 72 }}
+                      />
+                      <Typography.Text style={{ textAlign: 'center' }}>
+                        {app?.title}
+                      </Typography.Text>
+                    </Flex>
+                  </Col>
+                );
+              })}
+            </Row>
+          </>
+        ) : null}
+        <Form ref={formRef} layout="vertical">
+          <Row gutter={16}>
+            {/* @ts-ignore */}
+            {!globalThis.isElectron && baiClient._config.openPortToPublic ? (
+              <Col span={12}>
+                <Form.Item
+                  name={'clientIps'}
+                  label={
+                    <Flex gap={'xs'}>
+                      <Checkbox
+                        value={openToPublic}
+                        onChange={(value) =>
+                          setOpenToPublic(value.target.checked)
+                        }
+                      />
+                      {t('session.OpenToPublic')}
```

Comment:
Let's add information icon with simple description of what it is and what Allowed client IPs.
Here's the PR that how it works. (https://github.com/lablup/backend.ai-webui/pull/1178)
Probably this description may help.
> Open app to public:
Enabling this option allows application access for all users within the network. For access management, enter specific IP addresses in the 'Allowed IP' field. Leaving this field empty will permit access from all IP addresses. Consider setting IP restrictions when necessary for security management.